RELEASE.md

```diff
@@ -1,3 +1,10 @@
+# Release 0.1.6
+
+* New Module CloudTuner - Implementation of a library for hyperparameter tuning that is built into the KerasTuner and creates a seamless integration with Cloud AI Platform Optimizer Beta as a backend to get suggestions of hyperparameters and run trials. 
+* New application Monitoring - TensorFlow extension that exports its metrics to Stackdriver backend, allowing users to monitor the training and inference jobs in real time.
+* New experimental project Cloud_fit - an experimental module that enables training keras models on Google Cloud AI Platform by serializing the model, and datasets for remote execution.
```

Comment:
Google Cloud AI Platform? Cloud AI Platform?

Even thought it is implied, it's still good to drop link https://cloud.google.com/ai-platform/training/docs/overview

Reply:
Done, also updated to Cloud AI Platform